### ISSUE TITLE
Fix jsx-a11y eslint issue

### ISSUE THREAD

**Issue by msluther** [godaddy/react-img-carousel#47]
ESLint rules were updated to latest, which includes new rules for `jsx-a11y` to address accessibility.  Some rules were ignored to get the build passing.  These rules should be un-ignored and all issues should be fixed.

**Comment by thumbsupep** [comment]
I would like to work on this issue

**Comment by thumbsupep** [comment]
almost done. writing tests.
